pre-docs/installing_tasks_from_the_forge.md

```diff
@@ -52,12 +52,12 @@ This list of packaged modules is available in a Puppetfile in the [Bolt reposito
 
 -   [canary](https://github.com/puppetlabs/bolt/tree/master/modules/canary): Run action against a small number of nodes and only if it succeeds will it run on the rest.
 
--   [puppetdb\_fact](https://github.com/puppetlabs/bolt/tree/master/modules/puppetdb_fact): Collect facts for the specified nodes from the configured PuppetDB connection and use those facts to update the Target data types in a plan. 
```

Comment:
@melamos This page seems like it's a duplicate of bolt_installing_modules.md. Can it be deleted?

Reply:
This came up in the bolt channel regarding some broken links too. It seems like the long inaccurate slug is the one in use. Can we change that?

Reply:
@adreyer It looks like  installing_tasks_from_the_forge.md is what's on the docs site. From what I can tell, the duplicate (bolt_installing_modules) wasn't deleted when docs were refactored last summer. If I understand right, you want to keep bolt_installing_modules and get rid of installing_tasks_from_the_forge? I'm fine with that. Can you make the necessary change to pre-docs and I'll bring the docs site in line?

Reply:
done